### PR TITLE
Fix lods of effects

### PR DESCRIPTION
**Pull request by MadMaxFAF**
This branch is for bugfixes, general improvements, and new features. If your change is designed to alter balance, please make 
sure your changes are rebased onto [development/balance] (https://github.com/FAForever/fa/tree/development/balance), and target that branch with your PR.

### CODE REVIEW

**Comment by Garanas** [comment]
There is a syntax error:

image

File: effects/emitters/_test_explosion_a6_emit.bp [hunk 1]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 300
```
Comment (Garanas) [inline effects/emitters/_test_explosion_a6_emit.bp:14]: Needs a comma

Comment (MadMaxFAF) [reply effects/emitters/_test_explosion_a6_emit.bp]: fixed

**Review by Garanas — APPROVED** [review]
Looks good - nice touch on adding a comment that tells you where the emitter is used (or not at all).

**Comment by Garanas** [comment]
For future changes: fetch / pull in the latest version of `fix-lods-of-effects` of the faforever remote. Then before you make changes, create a new branch. Do your work on that branch. And then make a pull request for that branch into `fix-lods-of-effects`. 

With that you prevent these type of commits:
image

If you need help with this, just ping me here.

**Pull request merged**